### PR TITLE
docs(s3): update Bucket policy doc MTA-3844

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: macros/storage/important-bucket-policy.mdx
Comment (nerda-codes) [inline macros/storage/important-bucket-policy.mdx:9]:
```suggestion
    - The [owner](/identity-and-access-management/iam/concepts/#owner) of the Organization always has full rights and access to all resources and features in their Organization.
```

File: storage/object/api-cli/create-bucket-policy.mdx
Comment (nerda-codes) [inline storage/object/api-cli/create-bucket-policy.mdx:32]:
```suggestion
2. Open it in a code editor and paste the code below inside. This statement **only allows the specified user** to see the bucket and its objects.
```

File: storage/object/api-cli/create-bucket-policy.mdx
Comment (nerda-codes) [inline storage/object/api-cli/create-bucket-policy.mdx:56]:
```suggestion
3. Replace the `<USER_ID>` placeholder with the ID of the user to which you want to grant access. You can also grant access to an [application](/storage/object/api-cli/bucket-policy/#principal).
```

File: storage/object/api-cli/create-bucket-policy.mdx
Comment (nerda-codes) [inline storage/object/api-cli/create-bucket-policy.mdx:65]:
```suggestion
Make sure that you have [installed the AWS CLI](/storage/object/api-cli/object-storage-aws-cli/) before proceeding.
```

File: storage/object/concepts.mdx
Comment (nerda-codes) [inline storage/object/concepts.mdx:172]:
```suggestion
A principal is the target of a [bucket policy](#bucket-policy) and acquires the rights and permissions defined in the policy. The principal of a bucket policy can be an IAM user, or an IAM application. Bucket policies can have several principals attached to it.
```

**Review by nerda-codes — APPROVED** [review]

**Comment by SamyOubouaziz** [comment]
waiting for @crlptl to approve before merge

File: storage/object/api-cli/combining-iam-and-object-storage.mdx
Comment (crlptl) [inline storage/object/api-cli/combining-iam-and-object-storage.mdx:29]: We could say again here "at the project level". Even though it's written below, that's the main fact

File: storage/object/api-cli/combining-iam-and-object-storage.mdx
Comment (crlptl) [inline storage/object/api-cli/combining-iam-and-object-storage.mdx:105]: the first resource is useless here in this second statement

File: storage/object/api-cli/create-bucket-policy.mdx
Comment (ldecarvalho-doc) [inline storage/object/api-cli/create-bucket-policy.mdx:4]: it's the AWS CLi, no?

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**
